superset/db_engine_specs/base.py

```diff
@@ -638,8 +647,8 @@ def apply_limit_to_sql(
             sql = sql.strip("\t\n ;")
             qry = (
                 select("*")
-                .select_from(TextAsFrom(text(sql), ["*"]).alias("inner_qry"))
-                .limit(limit)
+                    .select_from(TextAsFrom(text(sql), ["*"]).alias("inner_qry"))
+                    .limit(limit)
```

Comment:
I expect these changes to cause linting errors

Reply:
Corrected this